common/middleware/src/meta_rules.rs

```diff
@@ -0,0 +1,710 @@
+use crate::{Middleware, Status};
+use http::types::body::{KeyValueMap, LineBufferMut};
+use lazy_static::lazy_static;
+use log::error;
+use regex::bytes::Regex as RegexB;
+use regex::Regex;
+use std::collections::HashMap;
+use std::ops::Deref;
+
+/// Env config options
+static LOGDNA_META_APP: &str = "LOGDNA_META_APP";
+static LOGDNA_META_HOST: &str = "LOGDNA_META_HOST";
+static LOGDNA_META_ENV: &str = "LOGDNA_META_ENV";
+static LOGDNA_META_FILE: &str = "LOGDNA_META_FILE";
+static LOGDNA_META_K8S_FILE: &str = "LOGDNA_META_K8S_FILE";
+static LOGDNA_META_JSON: &str = "LOGDNA_META_JSON";
+static LOGDNA_META_ANNOTATIONS: &str = "LOGDNA_META_ANNOTATIONS";
+static LOGDNA_META_LABELS: &str = "LOGDNA_META_LABELS";
+
+static K8S_LOG_DIR: &str = "/var/log/containers/";
+
+static MAP_PREFIX_ENV: &str = "env.";
+static MAP_PREFIX_ANN: &str = "annot.";
+static MAP_PREFIX_LAB: &str = "label.";
+
+lazy_static! {
```

Comment:
For reference [Once Cell](https://docs.rs/once_cell/latest/once_cell/) is a more modern replacement for the lazy_static macro, it's a little more generic and avoids a macro for the same functionality (It's also what's likely to be upstreamed into std eventually)